pkgs/development/python-modules/ckzg/default.nix

```diff
@@ -0,0 +1,48 @@
+{
+  lib,
+  fetchFromGitHub,
+  clang,
+  buildPythonPackage,
+  python,
+  pythonOlder,
+  pyyaml,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "ckzg";
+  version = "2.0.1";
+
+  src = fetchFromGitHub {
+    owner = "ethereum";
+    repo = "c-kzg-4844";
+    tag = "v${version}";
+    hash = "sha256-mpDVuAghIPplopr/mij9LSEjbVEwmZw3Agyp2DNBbNI=";
+    fetchSubmodules = true;
```

Comment:
Unvendor blst and update and include ours instead.

Reply:
I got from the previous PR. I will look into it refactoring it, another day.

Reply:
Also done!

Reply:
It seems not. Fixed!